pkg/ansible/runner/runner.go

```diff
@@ -297,7 +297,21 @@ func (r *runner) Run(ident string, u *unstructured.Unstructured, kubeconfig stri
 		if err != nil && err != http.ErrServerClosed {
 			logger.Error(err, "Error from event API")
 		}
+
+		// link the current run to the `latest` directory under artifacts
+		currentRun := filepath.Join(inputDir.Path, "artifacts", ident)
+		latestArtifacts := filepath.Join(inputDir.Path, "artifacts", "latest")
+		if _, err = os.Lstat(latestArtifacts); err == nil {
+			if err = os.Remove(latestArtifacts); err != nil {
+				logger.Error(err, "Error removing the latest artifacts symlink")
```

Comment:
This will happen the first time every time, think we can make this info and add context?

Reply:
Will it? I assumed the `os.Lstat` would return `err` if the file exists

Reply:
I just checked, if the file doesn't exist then an error will be returned by `os.Lstat`, so we won't make it to this part of the block.